README.md

```diff
@@ -1,118 +1,44 @@
-compiled.
-
-### Native installation
-
-#### Requirements
-
-Running the service natively on Windows is not supported, however it can be run using
-[WSL](https://msdn.microsoft.com/en-gb/commandline/wsl/about), in any
-case open up an Ubuntu prompt and install it from there.
-
-The majority of the installation is actually the dependencies, and there are better guides out there for
-each one, so install these and then follow the steps below. If you are not comfortable
-installing these yourself, use the Docker version.
-
-* [Ruby](https://www.ruby-lang.org/en/) (>= 2.4.x)
 * [Bundler](http://bundler.io/)
-* [Node](https://nodejs.org/en/) (>= 8.x though older should work)
-* [Yarn](https://yarnpkg.com/en/)
-* [Postgresql](https://www.postgresql.org/)
-
-#### Steps
-
-This guide is assuming you are using bash and installing on MacOS or Ubuntu.
-
-* Checkout the repo via git `git@github.com:barnardos/consent-form-builder-rails.git` or
-  [download it here](https://github.com/barnardos/consent-form-builder-rails/archive/master.zip).
-* Open a bash prompt in the project folder
-* Use `bundle install` to install all the Ruby dependecies
-* Use `yarn install` to install all the Node dependecies
-* To access postgres from terminal, type the command `psql postgres`
-* Add a user named 'consent' to your postgres instance, (command for this is `CREATE USER consent;`)
-* Add a database named 'consent' to your postgres instance, and make 'consent' the owner, (command for this is `CREATE DATABASE consent OWNER consent;`)
-* Add a database named 'consent_test' to your postgres instance, and make 'consent' the owner (command for this is `CREATE DATABASE consent_test OWNER consent_test;`)
+* [Hivemind](https://github.com/DarthSim/hivemind)
+* [Node](https://nodejs.org/en/)
+* [PostgreSQL](https://www.postgresql.org/)
+* [Ruby](https://www.ruby-lang.org/en/)
+* [Yarn](https://yarnpkg.com)
+
+Run these commands:
+
+* `bundle install` - install all Ruby dependencies
+* `yarn` - install all Javascript dependencies
+* Set up the Postgres database
+  * To access postgres from terminal, type the command `psql postgres`
+  * Add a user named 'consent' to your postgres instance
+    * `CREATE USER consent;`
+  * Add a database named 'consent' to your postgres instance, and make 'consent' the owner
+    * `CREATE DATABASE consent OWNER consent;`
+  * Add a database named 'consent_test' to your postgres instance, and make 'consent' the owner
+    * `CREATE DATABASE consent_test OWNER consent_test;`
 * Update the database schema using `bundle exec rails db:migrate`
-* If you have issues building your test database (`consent_test`), upgrade user to a SUPERUSER, access PostgreSQL in your terminal (`psql postgres`), then `ALTER USER consent WITH SUPERUSER;`. To check user attributes `\du`. Quit `\q`
-
-You can now start the rails server.
+* `hivemind Procfile.dev` - start a rails server with
```

Comment:
Start a rails server with `hivemind Procfile.dev` may sound a little clearer.

Reply:
For now, I am going to update this to read: